media/css/m24/components/footer-refresh.scss

```diff
@@ -316,21 +317,25 @@ $max-footer-content-width: $content-max;
     }
 
     .mzp-js-language-switcher-select {
-        background: $m24-color-alt-white url('/media/img/icons/m24-small/down-caret-white.svg') 95px 0.75em no-repeat;
+        @include bidi(((background-position, right 8px center, 8px center),));
+        background: $m24-color-alt-white url('/media/img/icons/m24-small/down-caret-white.svg') no-repeat;
         background-size: 16px 16px;
         border-radius: 0;
-        border: $border-width solid transparent;
-        color: $m24-color-black;
+        border: $border-width solid $m24-color-alt-black;
+        color: $m24-color-alt-black;
         font-weight: 600;
         margin-top: 16px;
-        min-width: unset;
         padding-left: 36px;
-        width: 136px;
+        max-width: 100%;
+        width: fit-content;
 
         &:hover,
         &:focus,
         &:focus-visible {
-            border-color: $m24-color-mid-gray;
+            color: $m24-color-white;
+            background: $m24-color-alt-black url('/media/img/icons/m24-small/up-caret.svg') no-repeat;
```

Comment:
**Nit:** (optional) Let's have the caret point down all the time to match the behaviour on the newsletter select boxes.

Reply:
Changing the caret to point up was specifically requested by Tess. This seems like a common behavior for a select. Should we update the newsletter select boxes to point up when open instead?

Reply:
It should at least point down until it is open, but I will discuss it with Tess.